lib/wrappers/libuv.nim

```diff
@@ -85,14 +86,14 @@ type
     evRENAME = 1,
     evCHANGE = 2
 
-  TFsEvent* {.pure, final, importc: "uv_fs_event_t", header: "uv.h".} = object
+  FsEvent* {.pure, final, importc: "uv_fs_event_t", header: "uv.h".} = object
     loop* {.importc: "loop".}: PLoop
     typ* {.importc: "type".}: HandleType
     close_cb* {.importc: "close_cb".}: CloseProc
     data* {.importc: "data".}: pointer
     filename {.importc: "filename".}: cstring
 
-  PFsEvent* = ptr TFsEvent
+  PFsEvent* = ptr FsEvent
```

Comment:
Wondered about those. The original issue was just for the `T` prefixes so I thought maybe there'd be a separate issue on its way. Either way, I can work on those as well this week.